packages/@react-types/datepicker/src/index.d.ts

```diff
@@ -133,9 +137,9 @@ interface SpectrumDatePickerBase<T extends DateValue> extends SpectrumDateFieldB
   createCalendar?: (identifier: CalendarIdentifier) => ICalendar
 }
 
-export interface SpectrumDatePickerProps<T extends DateValue> extends Omit<AriaDatePickerProps<T>, 'isInvalid' | 'validationState'>, SpectrumDatePickerBase<T> {}
+export interface SpectrumDatePickerProps<T extends DateValue> extends Omit<AriaDatePickerProps<T>, 'isInvalid' | 'validationState' | 'autoComplete'>, SpectrumDatePickerBase<T> {}
 export interface SpectrumDateRangePickerProps<T extends DateValue> extends Omit<AriaDateRangePickerProps<T>, 'isInvalid' | 'validationState'>, Omit<SpectrumDatePickerBase<T>, 'validate'> {}
-export interface SpectrumDateFieldProps<T extends DateValue> extends Omit<AriaDateFieldProps<T>, 'isInvalid' | 'validationState'>, SpectrumDateFieldBase<T> {}
+export interface SpectrumDateFieldProps<T extends DateValue> extends Omit<AriaDateFieldProps<T>, 'isInvalid' | 'validationState' | 'autoComplete'>, SpectrumDateFieldBase<T> {}
```

Comment:
Not supported in Spectrum? Only RAC?

Reply:
i'm fine with opening up to spectrum. wasn't sure if we wanted to keep this in rac for now

Reply:
I figured we just do in RAC since that's where it was asked for but it should be pretty easy to add to Spectrum if we want to